pkg/v1/cli/artifact/oci.go

```diff
@@ -17,6 +25,26 @@ func NewOCIArtifact(image string) Artifact {
 
 // Fetch an artifact.
 func (g *OCIArtifact) Fetch() ([]byte, error) {
-	// TODO(anujc25): implement OCI artifact fetch
-	return nil, nil
+	filesMap, err := carvelhelpers.GetFilesMapFromImage(g.Image)
+	if err != nil {
+		return nil, errors.Wrap(err, "unable fetch plugin binary")
+	}
+
+	var bytesData []byte
+	fileCount := 0
+
+	for path, fileData := range filesMap {
+		// Skip any testing related files if bundled
+		if strings.Contains(path, "test") {
+			continue
+		}
```

Comment:
**issue**: Sooner or later, it feels like this is going to have unintended consequences. For example, maybe someday we have `tanzu attestation` and we'll hit this check and `strings.contain("attestation", "test") == true`. Perhaps instead of just looking at `path` as a string, it'd be safer to parse `path` as a path and then look more precisely at the result?